support-models/src/main/scala/com/gu/support/acquisitions/AcquisitionData.scala

```diff
@@ -0,0 +1,70 @@
+package com.gu.support.acquisitions
+
+import com.gu.support.encoding.Codec
+import com.gu.support.encoding.Codec.deriveCodec
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class AbTest(
+  name: String,
+  variant: String
+)
+
+case class QueryParameter(
+  name: String,
+  value: String
+)
+
+/**
+ * Model for acquisition data passed by the referrer.
+ */
+case class ReferrerAcquisitionData(
+  campaignCode: Option[String],
+  referrerPageviewId: Option[String],
+  referrerUrl: Option[String],
+  componentId: Option[String],
+  componentType: Option[String],
+  source: Option[String],
+  abTests: Option[Set[AbTest]],
+  queryParameters: Option[Set[QueryParameter]],
+  hostname: Option[String],
+  gaClientId: Option[String],
+  userAgent: Option[String],
+  ipAddress: Option[String],
+  labels: Option[Set[String]]
+)
+
+object ReferrerAcquisitionData {
+  implicit val queryParameterDecoder = deriveDecoder[QueryParameter]
+  implicit val queryParameterEncoder = deriveEncoder[QueryParameter]
+  implicit val abTestDecoder = deriveDecoder[AbTest]
+  implicit val abTestEncoder = deriveEncoder[AbTest]
+
+  implicit val referrerAcquisitionDataDecoder: Decoder[ReferrerAcquisitionData] = deriveDecoder[ReferrerAcquisitionData]
+
+  implicit val referrerAcquisitionDataEncoder: Encoder[ReferrerAcquisitionData] = deriveEncoder[ReferrerAcquisitionData]
+}
+
+case class OphanIds(pageviewId: Option[String], visitId: Option[String], browserId: Option[String])
+
+object OphanIds {
+  import io.circe._
+  import io.circe.generic.semiauto._
+
+  implicit val decoder: Decoder[OphanIds] = deriveDecoder[OphanIds]
+
+  implicit val encoder: Encoder[OphanIds] = deriveEncoder[OphanIds]
+}
+
+case class AcquisitionData(
```

Comment:
from `support-models/src/main/scala/com/gu/support/workers/AcquisitionData.scala`
but this file is now more general than support-workers